jekyll/_cci2/skip-build.md

```diff
@@ -1,27 +1,35 @@
 ---
 layout: classic-docs
-title: Skip and Cancel Builds
-short-title: Skip and Cancel Builds
-description: How to Prevent CircleCI From Automatically Building
+title: Skip or Cancel Pipelines
```

Comment:
Can we avoid saying "pipelines" here? Maybe something like:

```suggestion
title: Skip and Cancel: avoiding unnecessary credit spend
```

Reply:
"Skip and cancel jobs and workflows"?

Reply:
Yeah, referring to jobs and workflows will be correct for now. We will probably eventually move the skip to pipeline level, but we can correct that when it happens.